src/restaurant_inspections_api/validations.clj

```diff
@@ -49,3 +51,16 @@
   [validated-map]
   {:invalid (into {} (filter #(false? (second %)) validated-map))
    :valid (into {} (filter #(not (false? (second %)))) validated-map) })
+
+(defn processable?
```

Comment:
hm.. processable is more related to an http behavior. That is, is the request processable? Inside we end up checking / running validations, but there's no reason why we could do something different from a validation in there.